### PR TITLE
Plugins should never fail the tests run

### PR DESCRIPTION
**Pull request by shay-stratoscale**

### CODE REVIEW

File: docker_test_tools/environment.py [hunk 1]
```diff
@@ -102,7 +105,10 @@ def teardown(self):
         log.debug("Tearing down the environment")
         try:
             for plugin in self.plugins:
-                plugin.stop()
+                try:
+                    plugin.stop()
+                except:
+                    logging.warning("Plugin %s failed, skipping", plugin)
```
Comment (eran-stratoscale) [inline docker_test_tools/environment.py]: logging.warning("Failed stopping Plugin %s, skipping", plugin)

File: docker_test_tools/environment.py [hunk 2]
```diff
@@ -88,7 +88,10 @@ def setup(self):
             self.run_containers()
 
             for plugin in self.plugins:
-                plugin.start()
+                try:
+                    plugin.start()
+                except:
+                    logging.warning("Plugin %s failed, skipping", plugin)
```
Comment (eran-stratoscale) [inline docker_test_tools/environment.py]: logging.warning("Failed starting Plugin %s, skipping", plugin)

File: docker_test_tools/stats.py [hunk 3]
```diff
@@ -139,43 +142,43 @@ def parse_line(self, line):
...
-
-        self.summary_data[name].update(
-            cpu_used=components['cpu'],
-            ram_used=components['ram'],
-            net_io_used=components['net'],
-            block_io_used=components['block']
-        )
-
-        return components
+        try:
```
Comment (eran-stratoscale) [inline docker_test_tools/stats.py]: How is this relevant to this commit?

Comment (shay-stratoscale) [reply docker_test_tools/stats.py]: this part use to fail, now it wont
